Helper/Data.php

```diff
@@ -635,6 +638,22 @@ public function getWsUsername($storeId = null)
 		return $wsUsername;
 	}
 
+	/**
+	 * @desc Retrieve the Live endpoint prefix key
+	 *
+	 * @param int|null $storeId
+	 * @return string
+	 */
+	public function getLiveEndpointPrefix($storeId = null)
```

Comment:
minor: shouldn't getLiveEndpointPrefix return always live_endpoint_url_prefix even if store is in demo mode?
This logic exists in L1187 anyway